test/recipes/03-test_fipsinstall.t

```diff
@@ -26,7 +26,9 @@ plan skip_all => "Test only supported in a fips build" if disabled("fips");
 
 # Compatible options for pedantic FIPS compliance
 my @pedantic_okay =
-    ( 'ems_check', 'no_drbg_truncated_digests', 'self_test_onload' );
+    ( 'ems_check', 'no_drbg_truncated_digests', 'self_test_onload',
+      'signature_digest_check'
```

Comment:
Why is this needed here?

Reply:
It's an extra options when installing the FIPS provider.
This makes an effort to check that the option is accepted.

Reply:
Are all options supposed to be here in general (Just a general question, not related to this PR)

Reply:
Dunno.  It seemed like a reasonable idea to check that the option is accepted.